### PR TITLE
Add support for i3-wm tiling manager.

### PR DESCRIPTION
**Pull request by akash0x53**

### CODE REVIEW

File: mackup/applications/i3wm.cfg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[application]
+name=i3wm
+
+[configuration_files]
+.i3/config
+.i3status.conf
+.xsession
```
Comment (lra) [inline mackup/applications/i3wm.cfg]: .xsession is not i3wm specific, it's more a X11 thing right?

Comment (akash0x53) [inline mackup/applications/i3wm.cfg]: Yeah, you're right, its not i3 specific, however its good to keep with i3wm configs.

Comment (lra) [inline mackup/applications/i3wm.cfg]: We can't have configs leak from one program to another, or every program will have all the configs. It should be in the X11 config.

**Comment by akash0x53** [comment]
Okay, removed .xsession.

**Comment by lra** [comment]
And you need to add yourself to the changelog ;)
https://github.com/lra/mackup/blob/master/CONTRIBUTING.md

**Pull request merged**